Sources/ComposableCoreLocation/Live.swift

```diff
@@ -74,6 +74,15 @@ extension LocationManager {
         .fireAndForget { dependencies[id]?.manager.requestWhenInUseAuthorization() }
       }
     #endif
+    
+    manager.requestTemporaryFullAccuracyAuthorization = { id, purposeKey in
+      #if (compiler(>=5.3) && !(os(macOS) || targetEnvironment(macCatalyst))) || compiler(>=5.3.1)
```

Comment:
I think you'll have to drop this condition too.

Reply:
Good shout, I'll also move the if #available to outside the closure.